website/content/en/highlights/2021-11-16-new-pipeline.md

```diff
@@ -0,0 +1,130 @@
+---
+date: "2021-11-16"
+title: "Pipelines is now available on Vector"
+description: "A guide for using the new pipelines functionality"
+authors: ["barieom"]
+pr_numbers: []
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: new feature
+---
+
+# Pipelines is now available on Vector
+
+We're excited to announce a new release that introduces the concept of Pipelines to Vector.
+
+Vector's topology is based on a [directed acylclic graph], enabling users to create a complex pipelines and powerful models that power advanced use cases, illustrated by the example diagram below:
+![DAG example](/img/vector-DAG-example.png)
+
+However, this is in contrast to many leading observability pipeline offerings — such as [Datadog's LWL Pipelines][] — by defining a linear set of transforms, in which the first transform feeds into the second, then the third, in sequential manner. Diagram below highlights the different flow compared to the DAG model:
+![New pipeline example](/img/new-pipeline-example.png)
+
+To align with industry comparable workflows, Vector now offers a linear transforms, reducing complexity. 
+
+To illustrate this better, let's assume we have a pipeline with three stages - source, transforms, then sink. The `transforms` directory has the following structure:
+
+```
+/transforms/
+└───processing/
+│   └───logs/   
+│       │   sudo.toml
+│       │   aws_s3_access_logs.toml
+│   
```

Comment:
expanding this in subfolders is not yet implemented 😉

Reply:
👍 Do you mean to say that users can only define pipelines as top-level configuration?

If so, I might be inclined to leave this highlight for 0.19.0 once we've implemented the subdirectory loading.

Reply:
@binarylogic - what're your thoughts?

Reply:
Agree, I would like to announce this once we support pipelines-per-file.

Reply:
> Do you mean to say that users can only define pipelines as top-level configuration?

Yes, only at the top level or in the `transforms` folder, but for now, you cannot split it more.